src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {
+      this.button.setAttribute('disabled', '');
+    }
   }
 
   public themeChanged(newValue: any) {
     this.styleEngine.applyTheme(this, newValue);
   }
 
+  public disabledChanged(newValue: any) {

```

Comment:
This appears to be called by the binding stuff within Aurelia.  I'm going to move this to `boolean | string` for now and align the if statement to match for clarity. If something else comes later on down the road we can re-visit this bit.